src/test/java/core/basesyntax/RegistratonServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistratonServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.PEOPLE.add(new User("user01", "password123", 25));
+    }
+
+    @Test
+    void register_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User(null, "password", 19)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("", "password", 19)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("short", "password", 20)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user01", "password", 20)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user06", null, 19)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user678", "", 20)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user678", "123", 20)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user678", "12345", 20)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user07", "password", null)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user123", "password123", 15)));
+        assertThrows(RegistrationException.class, () -> registrationService
+                .register(new User("user123", "password123", -5)));
+    }
```

Comment:
one @Test should cover one situation, e.g. null check, invalid age, invalid password, passwords are not same, and et cetera.

it makes your test class easy to understand.
Split one big test with a few small ones.
Remember about naming convention, e.g. `<methodUnderTest>_<state>_<expectedBehavior>`